### PR TITLE
feat: input with prefix component

### PR DESCRIPTION
**Pull request by alexbarnsley**
## Checklist

- [ ] Documentation _(if necessary)_
- [ ] Tests _(if necessary)_
- [ ] Ready to be merged

### CODE REVIEW

**Comment by ItsANameToo** [comment]
@alexbarnsley failing test

**Comment by alexbarnsley** [comment]
Fixed @ItsANameToo. Also added tests for the new input component

File: resources/views/inputs/input.blade.php [hunk 1]
```diff
@@ -1,22 +1,27 @@
...
+            @include('ark::inputs.includes.input-field', [
+                'name'         => $name,
+                'errors'       => $errors,
+                'id'           => $id ?? $name,
+                'inputClass'   => $inputClass ?? '',
+                'noModel'      => $noModel ?? false,
+                'model'        => $model ?? $name,
+                'keydownEnter' => $keydownEnter ?? null,
+                'max'          => $max ?? null,
+            ])
```
Comment (ItsANameToo) [inline resources/views/inputs/input.blade.php:23]: this used to have `{{ $attributes->except([...]) }}` so we could add any attribute we wanted without having to define it (e.g. `dusk` and whatnot). Will this still work when using this new approach? 🤔

Comment (alexbarnsley) [reply resources/views/inputs/input.blade.php:23]: Yeh. The attributes stuff is now in `input-field` and the way includes work is attributes are passed down anyway

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**